playwright/src/test/java/com/microsoft/playwright/TestPageEvaluate.java

```diff
@@ -645,4 +645,14 @@ void shouldAcceptParameter() {
     assertTrue(object instanceof Date);
     assertEquals(Date.from(instant), object);
   }
+
+  @Test
+  void shouldTransferMaps() {
+    assertEquals(mapOf(), page.evaluate("() => new Map([[1, { test: 42n }]])"));
```

Comment:
@yury-s Hi, does this mean playwright-java doesn't support transfering Map of Set??

Reply:
Solved with this comment https://github.com/microsoft/playwright-python/pull/2075#discussion_r1328099084,
thank you.